### PR TITLE
ss-306:  add UI for wallet settings page

### PR DESCRIPTION
**Pull request by vasylpopovych**
image

### CODE REVIEW

File: frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const InputsContainer: React.FC = () => {
+    const { control, errors } = useAppForm({
+        defaultValues: { name: '', balance: '' },
+    });
+
+    const currency = [
+        { value: 'USD', name: 'USD' },
+        { value: 'UAH', name: 'UAH' },
+    ];
+
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx:19]: We have API for currency, you can check an example in dashboard page

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]
Good, but please connect currency API

**Review by anton-otroshchenko — DISMISSED** [review]

File: frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx (hunk 1 above)
Comment (vasylpopovych) [reply frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx:19]: In this task, I only did UI. For the connection with the backend, I created another task and assigned it to myself. I will already connect the API there.

**Review by anton-otroshchenko — DISMISSED** [review]

File: frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    const handleDropdownChangeCurrency = useCallback(() => {
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx:20]: Why this function always return null? can we use variable for it?

Comment (vasylpopovych) [reply frontend/src/bundles/common/pages/wallet-settings/components/inputs-container/inputs-container.tsx:20]: I only did UI in this task. I created this function as a stub. For the connection to the backup, I created another task and assigned it to myself. I will release the entire functionality there.

**Review by Evcat6 — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Pull request merged**